website/client/components/inventory/equipment.vue

```diff
@@ -65,12 +65,15 @@
           v-for="(label, group) in gearTypesToStrings",
           :key="group",
           :item="flatGear[activeItems[group]]",
```

Comment:
is there a way to create a temporary variable inside `v-for` so that we don't need to write `flatGear[...` all the time ^^